psi4/src/psi4/libfock/MemDFJK.cc

```diff
@@ -125,5 +133,19 @@ int MemDFJK::max_nocc() const {
     }
     return max_nocc;
 }
+void MemDFJK::set_omega_alpha(double alpha) {
+    omega_alpha_ = alpha;
+    dfh_->set_omega_alpha(omega_alpha_);
+}
+void MemDFJK::set_omega_beta(double beta){
+    omega_beta_ = beta;
+    dfh_->set_omega_beta(omega_beta_);
+}
 void MemDFJK::set_do_wK(bool tf) { do_wK_ = tf; dfh_->set_do_wK(tf); }
+void MemDFJK::set_wcombine(bool wcombine) { 
+    wcombine_ = wcombine;
+    if (dfh_) {
+        dfh_->set_wcombine(wcombine); 
```

Comment:
Is there a reason `dfh_->set_wcombine` is conditional (`if (dfh_) {`) but `set_omega_alpha/beta` above are not?

Reply:
Yes.

I want the function to be able to be called before the JK object is initialized. The DFHelper object is constructed and initialized when the JK object is initialized, so an attempt to use this function without the conditional throws a seg fault.

Reply:
aha, thanks!